### ISSUE TITLE
Display events by time (i.e. now and next display) also before the conference starts

### ISSUE THREAD

**Issue by zuphilip** [Wilm0r/giggity#151]
It is IMO useful to have the events listed by time also for planning your attendance of a conference ahead. Such a list is already done by the "Now and Next" display. However, this is not available on non-conference days.

Can this view of the events by time been expanded such that it becomes also available on days before the conference (and afterwards)?

I would envision to still have the message "No events today" but then below an option to choose a specific conference day and then see all events at that day sort by the time.

**Comment by Wilm0r** [comment]
Hrm, so basically the timetable/tracks view but not grouped anymore, just one enormous list sorted by time?

(Sorry, no idea how I missed this so far!)

**Comment by zuphilip** [comment]
Yes, that is correct. Probably it is enough to just do something different in the case for "No events today": https://github.com/Wilm0r/giggity/blob/6dff8ab95205b349be015fedc56be64126fdb4c5/app/src/main/java/net/gaast/giggity/NowNext.java#L61-L63

**Comment by Wilm0r** [comment]
Hrm, so the "Now and Next" view wouldn't really work that way: It loads the schedule in "today's stuff only" mode, but indeed it could just not do so. But then still it's hardcoded to show, sorted by room order, the currently ongoing event for each room (if any), then the single next item for each. It's not chronological. Though.... I think there were requests to change that, too.

I was thinking searching for * (just star, no quotes) would be a better way to simulate this, but it doesn't seem to work. Putting the letter a in front of it works better, but yay completely random sorting.

The number of views is pretty large already so I'm not sure where to stop with adding options...

**Comment by zuphilip** [comment]
> The number of views is pretty large already so I'm not sure where to stop with adding options...

One possiblity would be to just use the same "Now and next" view. Since this view does not show anything for non-conference days and "next" could also been seen as "in some days next".

I don't know any details of these code, but maybe one could simply use the first conference day (and the earliest time) for the case that today is not a conference day. Maybe that could be an easy start to a feature here and one could leave open whether and how to continue for all events or some mechanism to go next after the first next etc.
